src/topology/vector_bundle/basic.lean

```diff
@@ -436,47 +378,50 @@ def coord_change (e e' : trivialization R F E) (b : B) : F ≃L[R] F :=
+     (e.linear_equiv_at R b (hb.1 : _)).symm.trans (e'.linear_equiv_at R b hb.2)
     else linear_equiv.refl R F }
 
-lemma coe_coord_change (e e' : trivialization R F E) {b : B}
+variables {R}
+
+lemma coe_coord_changeₗ (e e' : trivialization F (π E)) [e.is_linear R] [e'.is_linear R] {b : B}
   (hb : b ∈ e.base_set ∩ e'.base_set) :
-  ⇑(coord_change e e' b) = (e.linear_equiv_at b hb.1).symm.trans (e'.linear_equiv_at b hb.2) :=
+  ⇑(coord_changeₗ R e e' b)
+  = (e.linear_equiv_at R b hb.1).symm.trans (e'.linear_equiv_at R b hb.2) :=
 congr_arg linear_equiv.to_fun (dif_pos hb)
 
-lemma coord_change_apply (e e' : trivialization R F E) {b : B}
+lemma coord_changeₗ_apply (e e' : trivialization F (π E)) [e.is_linear R] [e'.is_linear R] {b : B}
   (hb : b ∈ e.base_set ∩ e'.base_set) (y : F) :
-  coord_change e e' b y = (e' (total_space_mk b (e.symm b y))).2 :=
+  coord_changeₗ R e e' b y = (e' (total_space_mk b (e.symm b y))).2 :=
 congr_arg (λ f, linear_equiv.to_fun f y) (dif_pos hb)
 
-lemma mk_coord_change (e e' : trivialization R F E) {b : B}
+lemma mk_coord_changeₗ (e e' : trivialization F (π E)) [e.is_linear R] [e'.is_linear R] {b : B}
   (hb : b ∈ e.base_set ∩ e'.base_set) (y : F) :
-  (b, coord_change e e' b y) = e' (total_space_mk b (e.symm b y)) :=
+  (b, coord_changeₗ R e e' b y) = e' (total_space_mk b (e.symm b y)) :=
 begin
   ext,
   { rw [e.mk_symm hb.1 y, e'.coe_fst', e.proj_symm_apply' hb.1],
     rw [e.proj_symm_apply' hb.1], exact hb.2 },
-  { exact e.coord_change_apply e' hb y }
+  { exact e.coord_changeₗ_apply e' hb y }
 end
 
 /-- A version of `coord_change_apply` that fully unfolds `coord_change`. The right-hand side is
 ugly, but has good definitional properties for specifically defined trivializations. -/
-lemma coord_change_apply' (e e' : trivialization R F E) {b : B}
+lemma coord_changeₗ_apply' (e e' : trivialization F (π E)) [e.is_linear R] [e'.is_linear R] {b : B}
   (hb : b ∈ e.base_set ∩ e'.base_set) (y : F) :
-  coord_change e e' b y = (e' (e.to_local_homeomorph.symm (b, y))).2 :=
-by rw [e.coord_change_apply e' hb, e.mk_symm hb.1]
+  coord_changeₗ R e e' b y = (e' (e.to_local_homeomorph.symm (b, y))).2 :=
+by rw [e.coord_changeₗ_apply e' hb, e.mk_symm hb.1]
 
-lemma coord_change_symm_apply (e e' : trivialization R F E) {b : B}
-  (hb : b ∈ e.base_set ∩ e'.base_set) :
-  ⇑(coord_change e e' b).symm = (e'.linear_equiv_at b hb.2).symm.trans (e.linear_equiv_at b hb.1) :=
+lemma coord_changeₗ_symm_apply (e e' : trivialization F (π E)) [e.is_linear R] [e'.is_linear R]
+  {b : B} (hb : b ∈ e.base_set ∩ e'.base_set) :
+  ⇑(coord_changeₗ R e e' b).symm
+  = (e'.linear_equiv_at R b hb.2).symm.trans (e.linear_equiv_at R b hb.1) :=
 congr_arg linear_equiv.inv_fun (dif_pos hb)
 
-end topological_vector_bundle.trivialization
+end trivialization
 
 end topological_vector_space
 
 section
-open topological_vector_bundle
 
 variables (B)
 variables [nontrivially_normed_field R] [∀ x, add_comm_monoid (E x)] [∀ x, module R (E x)]
```

Comment:
On line 436 there is still the definition `continuous_transitions`, that does not appear anywhere else. Do you still need it?

Reply:
I think this is indeed a relic of a previous approach, and can be removed.